### ISSUE TITLE
P4 Config file vs extension config

### ISSUE THREAD

**Issue by soraphis** [belkiss/niftyplugins#72]
I've configured the plugin to take my p4config.txt - as adviced.
But since the p4config.txt is checked into the project - as it should be (i guess!?) - it does not make sense to set the 'P4USER' there.

I'm also working on different depots with different usernames, so I can't just set it via the `p4 set` command.... 

I'm kinda confused what the best approach would be here, my initial idea would've been that it takes the values in the extension where the the sysvars are empty, but even that wouldn't really solve my problem, since the extension settings are not per project (i guess).

**Comment by belkiss** [comment]
Heya!

The p4config.txt should *not* be submitted to the repo, as you say, it contains per user/workspace info.

You just have to create one such file per workspace, adjust the info within, and that's it :)

Good luck, and let me know!

**Comment by soraphis** [comment]
> The p4config.txt should not be submitted to the repo, as you say, it contains per user/workspace info.

yeah, I just realized that. I wasn't aware that 'P4IGNORE' has actually a default value and thought I must include that line in the p4config for everyone to use a common ignore list.

thanks for answering so fast!
